rswag-specs/lib/rswag/specs/configuration.rb

```diff
@@ -7,63 +7,59 @@ def initialize(rspec_config)
         @rspec_config = rspec_config
       end
 
-      def swagger_root
-        ActiveSupport::Deprecation.warn('Rswag::Specs: WARNING: The method will be renamed to "openapi_root" in v3.0')
-        @swagger_root ||= begin
-          if @rspec_config.swagger_root.nil?
-            raise ConfigurationError, 'No swagger_root provided. See swagger_helper.rb'
+      def openapi_root
```

Comment:
@hasghari, the `swagger_root` in https://github.com/rswag/rswag/blob/8514da308f11d393fdf15c1e90be11061530a5d2/rswag-api/lib/rswag/api/configuration.rb#L4-L8 should be updated too